src/components/Pagination/Pagination.tsx

```diff
@@ -0,0 +1,69 @@
+import TransparentButton from "components/Buttons/TransparentButton";
+import "./Pagination.css";
+
+type Props = {
+  page: number;
+  pageCount: number;
+  onPageChange: any;
+};
+
+function range(start, end) {
+  let length = end - start + 1;
+  return Array.from({ length }, (_, idx) => idx + start);
+}
+
+function getPageNumbers(current, max = 1) {
+  if (max === 2) {
+    return [1, 2];
+  }
+  if (current <= 1) {
```

Comment:
how `current` can be less then 1? if it's current page number then the lowest value is 1?